mobify/platform-scaffold
# [Integration manager] demandware shipping

This PR adds the ability to submit shipping information in the checkout.

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1304
 **Linked PRs**: n/a

## Changes
- Renamed region and country fields in checkout forms
- Added DW connector command to get shipping method estimates
- Added DW connector command to get shipping data if user has already saved their shipping info
- Added DW connector command to submit shipping data

## How to test-drive this PR
- Run `npm run dev`
- Preview [the demandware site](https://preview.mobify.com/?url=http%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart
- navigate straight to https://mobify-tech-prtnr-na03-dw.demandware.net/checkout/ (don't go to the cart page since it hasn't been set up yet)
- confirm the page works as expected
- entering valid information and proceeding to the next page should work (the next page will be missing content)
